### ISSUE TITLE
User Story: admin can add users by username or email

### ISSUE THREAD

**Issue by allthedoll** [allthedoll/allspice-product#57]
## TL;DR

- As an admin I want to invite users by AllSpice username or their work email so that they can join my organization or team.

## Product requirements TL;DR

- [ ] Admin can add user by AllSpice username
- [ ] Admin can add user by email address

<img width="780" alt="Screenshot 2023-10-06 at 12 53 23 PM" src="https://github.com/allthedoll/allspice-product/assets/5758031/096b4444-c975-4932-b02d-a9a73c80b114">

## Design requirements

- TBD

## Technical requirements

- TBD

## Timeframe

- Noted in [onboarding initiative](https://docs.google.com/document/d/1N5YY-I72F4WMXZt2NSJVvB2eSypoLuMATuKbO_rVuQs/edit?usp=sharing)
